charmed_openstack_info/data/lp-builder-config/openstack.yaml

```diff
@@ -458,6 +494,97 @@ projects:
     charmhub: nova-compute-nvidia-vgpu
     launchpad: charm-nova-compute-nvidia-vgpu
     repository: https://opendev.org/openstack/charm-nova-compute-nvidia-vgpu.git
+    # TODO: charmhub migration is not completed yet for this charm.  It needs
+    # backporting to bionic (I think).
+    branches:
```

Comment:
this charm was released during the yoga cycle, it has no support for releases older than yoga.

Reply:
Maybe?  Judging by this commit https://opendev.org/openstack/charm-nova-compute-nvidia-vgpu/commit/8296660bb31eb5aa4f1be9bf7ad3a29678ff133a it looks like at least focal-ussuri should be supported?

Reply:
And previously, bionic-queens was supported: https://opendev.org/openstack/charm-nova-compute-nvidia-vgpu/commit/f8c5b6ceff7786f4127aac21449d60db556f24cf

Reply:
it has no branches though - https://github.com/openstack/charm-nova-compute-nvidia-vgpu/branches .

Reply:
Oh, I agree; hence I left it as a todo.  I guess we need to work out which ones are really needed and then update this as needed.